modale.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modale</title>
+</head>
+
+<style>
+    .modale{
+        display: flex;
+        flex-direction: column;
+        box-sizing: border-box;
+        padding:30px;
+        border: solid black;
+        /*width: 300px;
+        height: 100px;*/
+        position: absolute;
+        z-index: 2;
+        background-color: white;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 10px;
+        font-size: 120%;
+        font-weight: bold;
+    }
+
+    .sfondo{
+        position: fixed;
+        z-index: 1;
+        opacity: 0.2;
+    }
+/*
+     .indietro{
+        align-self: flex-end;
+        float: inline-end;
+        
+    }
+
+    .avanti{
+        align-self: flex-end;
+        float: inline-start;
+    }
+*/
+    .container{
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+
+</style>
+
+<body>
+    <p class="sfondo">
```

Comment:
transparent backgroudn of modal window (backdrop) should be another layer that contains the modal